Packs/Cryptocurrency/IndicatorFields/indicatorfield-Cryptocurrency_Address_Abuse_Type.json

```diff
@@ -0,0 +1,30 @@
+{
+    "id": "indicator_abusetype",
+    "version": -1,
+    "modified": "2020-12-16T11:52:18.044789Z",
+    "name": "Abuse Type",
+    "ownerOnly": false,
+    "cliName": "abusetype",
+    "type": "shortText",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 2,
+    "hidden": false,
+    "associatedTypes": [
+        "Cryptocurrency Address"
+    ],
+    "associatedToAll": false,
+    "unmapped": false,
+    "unsearchable": false,
+    "caseInsensitive": true,
+    "sla": 0,
+    "threshold": 72,
+    "fromVersion": "5.0.0"
```

Comment:
is there any reason why this should not be from version 5.5?

Reply:
not at all, changed to 5.5.0